### ISSUE TITLE
Identifying different link usages

### ISSUE THREAD

**Issue by egekorkan** [w3c/wot-thing-description#1947]
We are seeing that link has two different usages from TD designers:

1. Basic linking: linking to the UI, pdf, alternative representation, icon
2. "Semantic" linking: collection, controlledBy, tm:extends

This is annoying for TD consumers since they need to handle two styles very differently. If I am building a graph of related Things, I cannot blindly parse through the links.

Since these two different mechanisms, I think we can discuss splitting those.

**Comment by benfrancis** [comment]
This is not dissimilar to how HTML [uses](https://html.spec.whatwg.org/multipage/links.html#linkTypes) `<link>` tags. They may be used to link to other HTML documents (e.g. `<link rel="next">` and `<link rel="canonical">`), to alternate representations of an HTML document (e.g. `<link rel="alternate">`) or to related non-HTML resources (e.g. `<link rel="icon">`, `<link rel="manifest">`, `<link rel="stylesheet">`).

Is there a reason that a Consumer can not use the `rel` and `type` members to distinguish between different types of links, like those described in section [5.6.2](https://w3c.github.io/wot-profile/#sec-link-media-types) of WoT Profile? Did you have an alternative solution in mind?

**Comment by egekorkan** [comment]
I agree that it can be actually just done with more parsing. We should just mention the difference in the spec, thus the `Add to Documentation` label
